software/NRG_itho_wifi/01_HTML.ino

```diff
@@ -250,16 +250,16 @@ void handleDebug(AsyncWebServerRequest *request) {
   
   response->print(F("<br><span>File system: </span><span>"));
 #if defined (HW_VERSION_ONE)
-  SPIFFS.info(fs_info);
+  LITTLEFS.info(fs_info);
```

Comment:
Why not making a define for this?
Then you can switch between SPIFFS and LITTLEFS by editing a single line :)

Reply:
'#define SPIFFS LITTLEFS' could have done the trick as well I guess

Reply:
Nope, as that will backfire when you move back and include SPIFFS headers.

Better define it using a new define label.
`#define LITTLEFS ACTIVE_FS`

Reply:
> #define LITTLEFS ACTIVE_FS

Should be:
`#define ACTIVE_FS LITTLEFS`

I guess :)

Reply:
Yep :)